packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/studio/services/representations/DomainExplorerServices.java

```diff
@@ -90,16 +86,20 @@ public String getKind(Object self) {
         return kind;
     }
 
-    public boolean hasChildren(Object self) {
+    public boolean hasChildren(Object self, IEditingContext editingContext) {
```

Comment:
Pass the `IEditingContext` as the first parameter

Reply:
Done